deal.II-toolchain/packages/cmake.package

```diff
@@ -1,30 +1,27 @@
-MAJOR=3.16
-MINOR=3
+MAJOR=3.20
+MINOR=5
 VERSION=${MAJOR}.${MINOR}
 
-# We try to determine if we have a linux platform to use tarball install
-# instead of configure/make due to possible missing autotools on some
-# HPC clusters. We need to do this, because tarball install on MacOS or
-# other platforms is not maintainable by the candi authors.
+if [ -z ${CANDI_CMAKE_LOAD_TARBALL} ]; then
+    CANDI_CMAKE_LOAD_TARBALL=ON
+fi
```

Comment:
Maybe I dont understand... Where do you actually decide to use a tarball download? In the other packages the flag is hardcoded in the first line, e.g., adolc.package.

Reply:
this is for outside definitions, e.g.
export CANDI_CMAKE=OFF && ./candi.sh

Reply:
or if you put this to your local.cfg

Reply:
Shouldn't we be consistent and assume the tarball download for the user automatically? Means withouth the if block, only CANDI_CMAKE_LOAD_TARBALL=ON.

Reply:
it is the default. The example was with the opposite

Reply:
A OK, then I will take it on my list to update the others packages accordingly. Thanks for this hint!